target/linux/realtek/image/rtl8198c.mk

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: GPL-2.0-only
+
+define Device/f-secure_sense
+  LOADADDR := 0x80000000
+  LOADER_PLATFORM := rtl8198c
+  LOADER_TYPE := bin
+  LZMA_TEXT_START := 0x84000000
```

Comment:
```0x82000000``` (32MiB from ```0x80000000```) or ```0x81800000``` (24MiB from ```0x80000000```) may be enough, like ramips target.
https://github.com/openwrt/openwrt/pull/3279

Reply:
Understood. But will leave it as is because my test kernel + initramfs is quite big for now.